tests/components/velbus/test_binary_sensor.py

```diff
@@ -0,0 +1,29 @@
+"""Velbus binary_sensor platform tests."""
+
+from unittest.mock import patch
+
+from syrupy.assertion import SnapshotAssertion
+
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from . import init_integration
+
+from tests.common import MockConfigEntry, snapshot_platform
+
+
+async def test_entities(
+    hass: HomeAssistant,
+    snapshot: SnapshotAssertion,
+    config_entry: MockConfigEntry,
+    entity_registry: er.EntityRegistry,
+) -> None:
+    """Test all entities."""
+    with patch("homeassistant.components.velbus.PLATFORMS", [Platform.BINARY_SENSOR]):
+        await init_integration(hass, config_entry)
+
+    state = hass.states.get("binary_sensor.buttonon")
+    assert state
```

Comment:
This is now already implicitly tested by the snapshot, otherwise the snapshot call would fail